nodejs/lib/util/http.js

```diff
@@ -34,11 +34,11 @@ this.Http = (function() {
 					case 'application/x-msgpack':
 						body = msgpack.decode(body);
 				}
-				var error = body.error || {
-					statusCode: statusCode,
-					code: headers['x-ably-errorcode'],
-					message: headers['x-ably-errormessage']
-				};
+				var error = body.error ? ErrorInfo.fromValues(body.error) : new ErrorInfo(
+					headers['x-ably-errormessage'] || 'Error response received from server: ' + statusCode + ' body was: ' + Utils.inspect(body),
```

Comment:
If I'm right that this is client-side message synthesis, in lieu of a server header, is that specified anywhere? I can only find [HP7](https://docs.ably.io/client-lib-development-guide/features/#HP7). Is it strictly required as part of this fix?

Reply:
No, not strictly related to the other fix - this is just improving the user experience a bit when showing an errorinfo made from a response received from a customer's auth server (which won't populate ably-specific headers). (I added it for node just cause I noticed we already did something like it for browsers, so was inconsistent that we don't in node)

It's not actually necessary, and not specified anywhere, just a bit of polish